frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -357,7 +357,16 @@ export const WhatsNewMenu = (props: Props) => {
         heading={l10n.getString("whatsnew-holiday-promo-2023-news-heading")}
         image={HolidayPromo2023Hero}
         cta={
-          <Link href={yearlyPlanRefWithCoupon} legacyBehavior>
+          <Link
```

Comment:
Can we add a view-ping `ref` here?
```suggestion
          <Link
            ref=useGaViewPing({"category": "Holiday Promo News CTA", "label": "holiday-promo-2023-news-cta"})
```